testing_application/app.py

```diff
@@ -0,0 +1,82 @@
+from typing import TypedDict
+
+from robyn import Robyn, OpenAPI, Request, SubRouter
+from robyn.openapi import OpenAPIInfo, Contact, License, ExternalDocumentation, Components
+
+app: Robyn = Robyn(
+    file_object=__file__,
+    openapi=OpenAPI(
+        info=OpenAPIInfo(
+            title="Sample App",
+            description="This is a sample server application.",
+            termsOfService="https://example.com/terms/",
+            version="1.2.2",
```

Comment:
why do we have random version names here?

Reply:
ill change to v1.0.0~